app/exporters/formatters/manual_section_indexable_formatter.rb

```diff
@@ -1,4 +1,6 @@
-class ManualSectionIndexableFormatter
+require "formatters/abstract_indexable_formatter"
+
+class ManualSectionIndexableFormatter < AbstractIndexableFormatter
   def initialize(section, manual)
     @section = section
```

Comment:
I _think_ this could be `@entity = section` plus changes later on to use `entity` and not create a `attr_reader :section`, for more consistency with the other indexable formatters... But I assume there's a reason you chose not to do this already?